app/routes/post.js

```diff
@@ -29,7 +29,7 @@ router.patch(
 // DELETE A TASK
 router.delete(
```

Comment:
@tanujvyas10 To delete the user should be authenticated so add the middleware to check if authenticated or not.
you can use the middleware which is already written in the middleware folder.